app/presenters/concerns/assessment_tasks/assessment_detail_presenter.rb

```diff
@@ -46,16 +46,10 @@ def assessment_details_status_tag
       classes = ["#{govuk_tag_class} app-task-list__task-tag"]
 
       tag.strong class: classes do
-        status
+        I18n.t("assessment_details.#{status}")
```

Comment:
Alternatively we could change the status enum in the `AssessmentDetail` model to be "complete" instead of "completed". But I don't wand to get into a habit of making logic or data changes every time there's a change in the front end copy. Using translations seems simpler, because if tomorrow we decide that the tag should read "Done", "Finished" or "Banana" we only have to change it in one place!

Reply:
Yep agreed, nice approach 👍

Reply:
Agreed! Good choice 😃